src/NationalInstruments.Analyzers/Style/ChainOfMethodsWithLambdasAnalyzer.cs

```diff
@@ -36,25 +36,52 @@ public override void Initialize(AnalysisContext context)
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
 
             context.RegisterSyntaxNodeAction(
-                AnalyzeSyntaxNode,
+                AnalyzeSyntaxContext,
                 SyntaxKind.ExpressionStatement,
                 SyntaxKind.EqualsValueClause,
                 SyntaxKind.Argument,
                 SyntaxKind.ArrowExpressionClause);
+
+            context.RegisterSyntaxNodeAction(
+                AnalyzeArrayInitializerContext,
+                SyntaxKind.ArrayInitializerExpression);
         }
 
-        private static void AnalyzeSyntaxNode(SyntaxNodeAnalysisContext context)
+        private static void AnalyzeSyntaxContext(SyntaxNodeAnalysisContext context)
         {
             var parentSyntaxNode = context.Node;
 
             // Find the invocation expression i.e
             // a method/delegate call or a property access
             // or a chain of them
             var invocationExpressionSyntax = parentSyntaxNode
-                .DescendantNodes()
+                .DescendantNodes(IsNotArrayCreationSyntax)
                 .OfType<InvocationExpressionSyntax>()
                 .FirstOrDefault();
 
+            AnalyzeSyntaxNode(invocationExpressionSyntax, context.ReportDiagnostic);
+
+            bool IsNotArrayCreationSyntax(SyntaxNode syntaxNode) => !syntaxNode.IsKind(SyntaxKind.ArrayCreationExpression);
+        }
+
+        private static void AnalyzeArrayInitializerContext(SyntaxNodeAnalysisContext context)
+        {
+            var arrayInitializerSyntaxNode = context.Node;
```

Comment:
Drop the "node" suffix (the other analyzers don't call them nodes)

Reply:
Fixed [here](https://github.com/ni/csharp-styleguide/pull/65/commits/f0828a0e8908b2cfaac4199c0551a709ecdeb0df).